coops-ui/src/components/AnnualReport/Directors.vue

```diff
@@ -740,11 +743,15 @@ export default class Directors extends Mixins(DateMixin, ExternalMixin) {
     this.draftDate = date
   }
 
+  public getOriginalDirectors () {
```

Comment:
This should be private unless you REALLY need to call it externally (which I would discourage since we should be using props/events to control components).

Reply:
Done.